### DISCUSSION TITLE
2024/03/short-videos/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/xiangyun#100] [General]
# 2024/03/short-videos/

https://xiangyun.rbind.io/2024/03/short-videos/

**Comment by XiangyunHuang** [comment]
为了分析「燕麦行游」的商业模式，我看了她好多视频。春天来了，我跟着做一道凉拌折耳根。在美团买菜上买的折耳根（应该是标准化流程清洗过的），和她的不一样，没有叶子（有叶的话平台恐不好保存），根很长，不好洗，味道不够冲，少了点菜味。这道菜在家乡的大自然中吃起来比在钢铁森林里要有烟火气。没有野葱用小葱替代，另外加了香菜，用老干妈风味豆酱拌（如果有合适的辣酱就更好），再放了点盐，我感觉做的还行，连拌了两顿。

**Reply by yihui** [reply]
我不是很明白拌个折耳根难道不是闭着眼睛瞎拌就可以了么，为啥还需要看别人的食谱嘞（让我想起有人煮菱角看菜谱之事）。这玩意儿就主打一个冲，折耳根本身不够劲爆的话，再好的调料都没用，葱、香菜、豆瓣酱那些调料在折耳根面前不都黯然失色么。你可以考虑买个土盆自己种一盆的，从老家挖一把来就行了。我放在地下室过冬的折耳根已经在冒新芽了。

**Reply by XiangyunHuang** [reply]
我是看了很多她的视频才去拌折耳根的，而不是为了拌折耳根去看她的视频。看看平台和她提供的菜在质量、价格方面的差别，以前没注意这些。这次主要是为了了解短视频这个新产品带来的新商业模式。

**Reply by yihui** [reply]
了然。
